torntools/scripts/js/globalFunctions.js

```diff
@@ -1615,6 +1615,7 @@ const STORAGE = {
 			stats_estimate: {
 				global: true,
 				profile: true,
+				attack_page: true,
```

Comment:
Not sure it should be on by default. Let's ask @Mephiles.

Reply:
Changed to off by default.